### PR TITLE
[Layout] Create Login Page

### PR DESCRIPTION
**Pull request by kaz-m-93**
create login page

### CODE REVIEW

File: app/assets/stylesheets/shared.scss [hunk 1]
```diff
@@ -41,6 +41,19 @@ body {
...
+		margin: auto;
+	
+		h1 {
+			text-align: center;
+		}
+	
+		.btn {
+			margin-top: 10px;
+		}
+	}
```
Comment (CharlotteDS) [inline app/assets/stylesheets/shared.scss:56]: Please use 2 spaces for indentation in `.scss` files.

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class SessionsController < ApplicationController
```
Comment (CharlotteDS) [inline app/controllers/sessions_controller.rb:1]: Please use 2 spaces for indentation in `.rb` files.

File: app/controllers/sessions_controller.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+    def new
+        @user = User.new
```
Comment (CharlotteDS) [inline app/controllers/sessions_controller.rb:3]: No need to pass `@user = User.new` in the new method for login.

**Review by CharlotteDS — CHANGES_REQUESTED** [review]
Please find and remove all `.DS_Store` files from your application before pushing.

**Comment by kaz-m-93** [comment]
i fixed indentation, deleted content of new method and modified content in formfor

**Review by CharlotteDS — APPROVED** [review]

**Pull request merged**